doc/user/content/guides/cdc-postgres.md

```diff
@@ -0,0 +1,260 @@
+
+    ```json
+    {
+        "name": "your-connector",
+        "config": {
+            "connector.class": "io.debezium.connector.postgresql.PostgresConnector",
+            "tasks.max": "1",
+            "plugin.name":"pgoutput",
+            "database.hostname": "postgres",
+            "database.port": "5432",
+            "database.user": "postgres",
+            "database.password": "postgres",
+            "database.dbname" : "postgres",
+            "database.server.name": "pg_repl",
+            "table.include.list": "public.table1",
+            "publication.autocreate.mode":"filtered",
+            "key.converter": "io.confluent.connect.avro.AvroConverter",
+            "value.converter": "io.confluent.connect.avro.AvroConverter",
+            "value.converter.schemas.enable": false
+        }
+    }
+    ```
+
+    You can read more about each configuration property in the [Debezium documentation](https://debezium.io/documentation/reference/1.6/connectors/postgresql.html#postgresql-connector-properties). By default, the connector writes events for each table to a Kafka topic named `serverName.schemaName.tableName`.
+
+
+1. Start the Debezium Postgres connector using the configuration file:
+
+    ```bash
+    export CURRENT_HOST='<your-host>'
+
+    curl -i -X POST -H "Accept:application/json" -H  "Content-Type:application/json" \
+    http://$CURRENT_HOST:8083/connectors/ -d @register-postgres.json
+    ```
+
+1. Check that the connector is running:
+
+    ```bash
+    curl http://$CURRENT_HOST:8083/connectors/inventory-connector/status
+    ```
+
+    The first time it connects to a Postgres server, Debezium takes a [consistent snapshot](https://debezium.io/documentation/reference/1.6/connectors/postgresql.html#postgresql-snapshots) of the tables selected for replication; so, you should see that the pre-existing records in the replicated table are initially pushed into your Kafka topic:
+
+    ```bash
+    /usr/bin/kafka-avro-console-consumer \
+      --bootstrap-server kafka:9092 \
+      --from-beginning \
+      --topic pg_repl.public.table1
+    ```
+
+### Create a source
+
+Debezium emits change events using an envelope that contains detailed information about upstream database operations, like the `before` and `after` values for each record. To create a source that interprets the [Debezium envelope](/sql/create-source/avro-kafka/#debezium-envelope-details):
+
+```sql
+CREATE SOURCE kafka_repl
+FROM KAFKA BROKER 'kafka:9092' TOPIC 'pg_repl.public.table1'
+KEY FORMAT AVRO USING CONFLUENT SCHEMA REGISTRY 'http://schema-registry:8081'
+VALUE FORMAT AVRO USING CONFLUENT SCHEMA REGISTRY 'http://schema-registry:8081'
+ENVELOPE DEBEZIUM UPSERT;
+```
+
+Enabling `UPSERT` will only work for tables with `REPLICA IDENTITY DEFAULT`/`INDEX` and, although this approach is less resource-intensive for the upstream database, it will require more memory in Materialize.
```

Comment:
Technically it is only required for those replica identities, `DEBEZIUM UPSERT` should work fine with replica identity full, it's just kind of a waste unless you happen to know that your data in Kafka is corrupted (e.g. because of duplicate or missing rows).

This is accurate as written. If we want to provide slightly more guidance we could mention that Debezium Upsert needs to track state proportional to the number of unique primary keys in the changing data, so Debezium Upsert works best when there are a small number of quickly changing rows, and REPLICA IDENTITY FULL works best for pretty much everything else.

Reply:
That was my understanding, but re-reading it now it's not as clear as you put it. I'll adapt! Thanks again!